llap-server/src/java/org/apache/hadoop/hive/llap/io/api/impl/LlapIoImpl.java

```diff
@@ -150,6 +151,8 @@ private LlapIoImpl(Configuration conf) throws IOException {
       LowLevelCachePolicy
           realCachePolicy =
           useLrfu ? new LowLevelLrfuCachePolicy(minAllocSize, totalMemorySize, conf) : new LowLevelFifoCachePolicy();
+      // TODO: if realCachePolicy is not something that supports proactive caching
```

Comment:
nit: `TODO: if realCachePolicy is not something that supports proactive caching`
Maybe:
`TODO: if realCachePolicy is not something that supports proactive cache eviction`?

Reply:
Agreed - however with the PR being approved and having green tests I won't change just this comment, it will be removed with the next patch in line (https://issues.apache.org/jira/browse/HIVE-24341) anyway, very shortly :)